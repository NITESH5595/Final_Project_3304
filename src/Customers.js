import React, { useState } from "react";
import { signInWithEmailAndPassword } from "firebase/auth";
import "./index.css";
import { useNavigate } from "react-router-dom";
import { auth } from "./base";

function Customers() {
  const navigate = useNavigate();
  const goBack = () => {
    navigate("/");
  };
  return (
    <div className="mainContainer">
      <div className="loginForm">
        <p className="paragraph">This is our customers Page </p>
        <br />
        <button className="btn" onClick={() => goBack()}>
          Go Back
        </button>
      </div>
    </div>
  );
}

export default Customers;
