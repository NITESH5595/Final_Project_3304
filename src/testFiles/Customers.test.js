import React from "react";
import renderer from "react-test-renderer";
import Customers from "../Customers";

describe("App", () => {
  it("renders without crashing given the required props", () => {
    // const props = {

    const wrapper = renderer.create(<Customers />).toJSON();
    expect(wrapper).toMatchSnapshot();
  });
});
