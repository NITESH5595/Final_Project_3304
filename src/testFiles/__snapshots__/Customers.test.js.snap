// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders without crashing given the required props 1`] = `
<div
  className="mainContainer"
>
  <div
    className="loginForm"
  >
    <p
      className="paragraph"
    >
      This is our customers Page 
    </p>
    <br />
  </div>
</div>
`;
